server/scripts/get-email-list.js

```diff
@@ -0,0 +1,18 @@
+const { dbClient } = require('../db');
+require('dotenv').config({ path: '../.env' });
+(async function exec(){
+    await dbClient.connect();
+    const res = await dbClient.db.query(
+   `SELECT DISTINCT body->> 'emailAddress' AS email 
+    FROM participants p 
+    JOIN participants_status ps 
+        ON  p.id = ps.participant_id AND ps.status != 'hired'
+    WHERE 
+        CAST(p.body ->> 'interested' AS TEXT) = 'yes'  
```

Comment:
is p.body->>'interested'::text = 'yes' another option instead of using the CAST function? I think they're equivalent, I just haven't seen this one before.